### PR TITLE
gamenet: Add 0.7 server info fields

### PR DESCRIPTION
**Pull request by ChillerDragon**
Generated using

```bash
cd gamenet/generate
python3 serialize.py --version 0.7 raw/teeworlds-0.7.5.py > spec/teeworlds-0.7.5.json
cd ../../
./generate_all
```

Tested using the wireshark dissector

image

### CODE REVIEW

File: gamenet/generate/msg_system.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
     ( 1, "info", "vital", "s:version s?:password i?:client_version"),
     ( 2, "map_change", "vital", "s:name i:crc i:size i:num_response_chunks_per_request i:chunk_size h:sha256"),
     ( 3, "map_data", "vital", "r:data"),
-    ( 4, "server_info", "vital", "r:data"),
+    ( 4, "server_info", "vital", "s:version s:name s:hostname s:map_name s:game_type i:flags i:skill_level i:player_count i:player_slots i:client_count i:max_clients"),
```
Comment (heinrich5991) [inline gamenet/generate/msg_system.py:73]:
```suggestion
    ( 4, "server_info", "vital", "s:version s:name s:hostname s:map s:game_type i:flags i:skill_level i:num_players i:max_players i:num_clients i:max_clients"),
```

**Review by heinrich5991 — CHANGES_REQUESTED** [review]
Thanks for the PR. It looks good to me. In order to match the existing serverinfo fields in `gamenet`, could you rename them as I suggested?

**Review by heinrich5991 — APPROVED** [review]
I generated the data again (seems to have been forgotten) and squashed the commits. Does it also look good to you now, @ChillerDragon?

**Comment by ChillerDragon** [comment]
didn't test but LGTM

Yeah I just clicked the GitHub button to commit your changes 🙈 thats why the generate was missing.

**Pull request merged**